examples/tutorials/Introduction to Langrangian Neural Networks.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<p>Train model for a lesser number of epochs rather than stopping training mid way. Loss seems to reduce well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3905/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>